docs/adr/0001-daily-active-use-server-side-metrics-glean.md

```diff
@@ -0,0 +1,114 @@
+# Measuring Server-Side Daily Active Use (DAU) With Glean
+
+* Status: proposed
+* Deciders: Taddes Korris, David Durst, JR Conlin, Phillip Jenvey
+* Date: 2024-10-09
+
+Technical Story: 
+[Jira Epic Url](https://mozilla-hub.atlassian.net/browse/SYNC-4389)
+
+
+## Context and Problem Statement
+
+There is a requirement to move away from the current measurement of Sync Daily Active Use (DAU), which measures usage via FxA/Mozilla Accounts.
+The addition of Relay, device backups/migrations, through FxA means the metric will no longer an be accurate reflection of Sync usage. This is due to increased browser sign-ins with the potential of overcounting, or not counting those logging into the browser without Sync enabled. Furthermore, there is a broader organizational movement towards measuring DAU within services themselves. 
+
+The goal is to measure DAU (and subsequently WAU & MAU) by emitting metrics from syncserver-rs itself. This requires the following data:
+* User identifier (hashed_fxa_uid)
+* Timestamp
+* Platform (from UserAgent: Desktop, Fenix, iOS)
+* Collection updated
```

Comment:
We only came to this conclusion earlier today but I'll note it's looking like we'll wanna remove this line (and the according entry in the schema) at some point